pkg/api/unversioned/group_version.go

```diff
@@ -179,6 +179,18 @@ func (gv GroupVersion) String() string {
 	return gv.Version
 }
 
+// VersionForGroupKind identifies the preferred GroupVersion for a GroupKind, or false if no
+// group version is preferred.
+func (gv GroupVersion) VersionForGroupKind(group GroupKind) (GroupVersion, bool) {
```

Comment:
Bad name.  `groupKind` instead of just `group`?  Something with kind in the name anyway.